test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+      toChainId: toChainId,
+      fromTokenAddress: fromTokenAddress,
+      toTokenAddress: toTokenAddress,
+      fromAmount: fromAmount,
+      serviceProvider: CrossChainServiceProvider.SocketV2, // Optional parameter
+    };
+    // console.log(quoteRequestPayload);
+
+    try {
+      assert.isNumber(
+        quoteRequestPayload.fromChainId,
+        "The fromChainId value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNumber(
+        quoteRequestPayload.toChainId,
+        "The toChainId value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.fromTokenAddress,
+        "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8",
+        "The fromTokenAddress value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.toTokenAddress,
+        "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
+        "The toTokenAddress value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        quoteRequestPayload.fromAmount,
+        "The fromAmount value is not displayed in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        quoteRequestPayload.serviceProvider,
+        "SocketV2",
+        "The serviceProvider value is not displayed correct in the quoteRequest Payload."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    // GET THE CROSS CHAIN QUOTES
+    let batchCrossChainTransaction;
+    let quotes = await sdkMainNet.getCrossChainQuotes(quoteRequestPayload);
+    // console.log("Quotes::: ", quotes);
+
+    try {
+      assert.strictEqual(
+        quotes.items[0].provider,
+        "socketv2",
+        "The provider value is not displayed correct in the quotes response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        quotes.items[0].approvalData,
+        "The approvalData value is not displayed correct in the quotes response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        quotes.items[0].transaction,
+        "The transaction value is not displayed correct in the quotes response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        quotes.items[0].estimate,
+        "The estimate value is not displayed correct in the quotes response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    if (quotes.items.length > 0) {
+      // Select the first quote
+      let quote = quotes.items[0];
+      // console.log("Quote Selected: ", quote);
```

Comment:
Please remember to remove any dead code, or leave a comment into why it's here.

Reply:
Removed the commented code from the test cases.